src/bindgen/utilities.rs

```diff
@@ -400,6 +395,16 @@ impl SynAbiHelpers for syn::Abi {
             false
         }
     }
+    fn is_cmse(&self) -> bool {
+        if let Some(ref lit_string) = self.name {
+            matches!(
+                lit_string.value().as_str(),
+                "cmse-nonsecure-entry" | "cmse-nonsecure-call"
```

Comment:
Hmm, I'm not sure why these function shouldn't be tagged with `__attribute__((cmse_nonsecure_call))` and `__attribute__((cmse_nonsecure_entry))`, respectively. Mind elaborating on that?